src/index.js

```diff
@@ -3,19 +3,25 @@ const path = require('path');
 const { Calculator } = require('fqm-execution');
 const { Workbook } = require('exceljs');
 const { parseQueryFilters } = require('./util/parseQueryFilters');
-const mustSupports = require('./mustSupports.json');
+const qicoreMustSupports = require('./qicore-must-supports.json');
+const uscoreMustSupports = require('./uscore-must-supports.json');
 
 if (!fs.existsSync('connectathon')) {
   console.error('Error: could not find "connectathon" directory. Ensure you run ./setup.sh first');
   process.exit(1);
 }
 
-if (!fs.existsSync('src/mustSupports.json')) {
-  console.error('Error: could not find "src/mustSupports.json". Ensure you run ./setup.sh first');
+if (!fs.existsSync('./src/qicore-must-supports.json')) {
```

Comment:
If I remove the `qicore-must-supports.json` file, I never actually get this `console.error` because I get a `MODULE_NOT_FOUND` error from the earlier `require` statement. If we want to keep these `existSync` checks, maybe we should remove the `require` statements for the MS files and read them in using fs after checking that they exist

Reply:
good call